chanfitness/chaneventstore.go

```diff
@@ -226,7 +226,10 @@ func (c *ChannelEventStore) Stop() error {
 
 	// Stop the ticker after the goroutine reading from it has exited, to
 	// avoid a race.
-	c.cfg.FlapCountTicker.Stop()
+	if c.cfg.FlapCountTicker == nil {
+		return fmt.Errorf("ChannelEventStore FlapCountTicker not " +
+			"initialized")
+	}
```

Comment:
still need to stop it if not nil though 🤓

Reply:
great catch 😅